ivi/interface/pyvisa.py

```diff
@@ -112,20 +113,20 @@ def trigger(self):
 
     def clear(self):
         "Send clear command"
-        raise NotImplementedError()
+        self.instrument.clear()
 
     def remote(self):
         "Send remote command"
-        raise NotImplementedError()
+        self.instrument.gpib_control_ren(pyvisa.constants.VI_GPIB_REN_ASSERT_ADDRESS)
```

Comment:
I was not able to find a more general VISA function for this purpose. Can you point me to one?